packages/wallet/api/src/marshal-contexts.js

```diff
@@ -324,6 +324,24 @@ export const makeImportContext = (makePresence = defaultMakePresence) => {
   };
 
   return harden({
+    /**
+     * @param {BoardId} id
+     * @param {unknown} val
+     */
+    initBoardId: (id, val) => {
+      initSlotVal(boardObjects, id, val);
+    },
+    /**
+     * @param {BoardId} id
+     * @param {unknown} val
+     */
+    ensureBoardId: (id, val) => {
+      if (boardObjects.byVal.has(val)) {
+        assert.equal(boardObjects.byVal.get(val), id);
+        return;
+      }
+      initSlotVal(boardObjects, id, val);
+    },
     fromMyWallet: Far('wallet marshaller', { ...marshal.fromMyWallet }),
```

Comment:
This things ends up with both methods and properties?  Either that or I'm confused about what gets wrapped with `far` here.

Reply:
yeah this is mixing, which is okay because the object returned isn't remote